benchmark/src/bin/ruben.rs

```diff
@@ -67,8 +68,7 @@ mod tests {
     rusty_fork_test! {
         #[test]
         fn test_benchmarker_counters() {
-            let (faucet_account_keypair, faucet_key_file_path, _temp_dir) =
-                generate_keypair::load_faucet_key_or_create_default(None);
+            let (faucet_account_keypair, faucet_key_file_path, _temp_dir) = generate_keypair::load_faucet_key_or_create_default(None);
```

Comment:
Does it let you do such long lines without complaining?

Reply:
It seems the checks are only checking the squashed commit, I mean the entire PR.